shared/validation/place-order-request-validator.ts

```diff
@@ -77,6 +78,24 @@ function isNullOrWhitespace(input: string) {
     return !input || input.replace(/\s/g, '').length < 1;
 }
 
+function validateOrderTime(): string[] {
```

Comment:
There are no tests for this validation. I think this validation needs tests.

Also, the errors should all be the same error message `Sorry, the shop is now closed`. If this error is present, no other error should be shown (i.e. it should be the first validation to run and if it doesn't pass then no other validations should run.